cdap-docs/cdap-apps/source/hydrator/creating.rst

```diff
@@ -193,7 +193,275 @@ property replaces the ``schedule`` property of an ETL Batch Application.
+                \"fields\":[
+                  {\"name\":\"userId\",\"type\":\"string\"},
+                  {\"name\":\"purchaseItem\",\"type\":\"string\"},
+                  {\"name\":\"purchasePrice\",\"type\":\"long\"}
+                ]
+              }",
+              "name": "testStream",
+              "duration": "1d"
+            }
+          }
+        },
+        "sinks": [
+          {
+            "name": "replicaTable",
+            "plugin": {
+              "name": "Table",
+              "properties": {
+                "schema": "{
+                  \"type\":\"record\",
+                  \"name\":\"etlSchemaBody\",
+                  \"fields\":[
+                    {\"name\":\"userId\",\"type\":\"string\"},
+                    {\"name\":\"purchaseItem\",\"type\":\"string\"},
+                    {\"name\":\"purchasePrice\",\"type\":\"long\"}
+                  ]
+                }",
+                "name": "replicaTable",
+                "schema.row.field": "userId"
+              }
+            }
+          },
+          {
+            "name": "usersTable",
+            "plugin": {
+              "name": "Table",
+              "properties": {
+                "schema": "{
+                  \"type\":\"record\",
+                  \"name\":\"etlSchemaBody\",
+                  \"fields\":[
+                    {\"name\":\"userId\",\"type\":\"string\"}
+                  ]
+                }",
+                "name": "targetCustomers",
+                "schema.row.field": "userId"
+              }
+            }
+          }
+        ],
+        "transforms": [
+          {
+            "name": "spendingUsersScript",
+            "plugin": {
+              "name": "Script",
+              "properties": {
+                "script": "function transform(input, context) {
+                            if (input.purchasePrice > 1000) {
+                              return {'userId' : input.userId};
+                            }
+                          }",
+                "schema": "{
+                  \"type\":\"record\",
+                  \"name\":\"etlSchemaBody\",
+                  \"fields\":[
+                    {\"name\":\"userId\",\"type\":\"string\"}
+                  ]
+                }"
+              }
+            }
+          }
+        ],
+        "connections": [
+          {
+            "from": "purchaseStats",
+            "to": "replicaTable"
+          },
+          {
+            "from": "purchaseStats",
+            "to": "spendingUsersScript"
+          },
+          {
+            "from": "spendingUsersScript",
+            "to": "usersTable"
+          }
+        ],
+        "schedule": "* * * * *",
+        "engine": "mapreduce"
+      }
+    }
+
+**Note:**
+Pipeline connections can be configured to fork from a stage, with the output of the stage sent to
+two or more configured stages; in the above example, the same output record is sent to the other stages.
```

Comment:
"in the above example, the same output record is sent to the other stages."  This statement doesn't really clarify anything to me.

What we're really trying to communicate is that records output by a stage are sent to all stages connected to it. In the above example, a record output by the 'purchaseStats' stage will get sent to both the 'replicaTable' and 'spendingUsersScript' stages.